wicket/src/ui/panes/update.rs

```diff
@@ -218,6 +240,24 @@ impl UpdatePane {
         match self.popup.as_ref().unwrap() {
             PopupKind::Logs => None,
             PopupKind::MissingRepo => None,
+            PopupKind::Ignition => match cmd {
+                Cmd::Up => {
+                    self.ignition.key_up();
+                    Some(Action::Redraw)
+                }
+                Cmd::Down => {
+                    self.ignition.key_down();
+                    Some(Action::Redraw)
+                }
+                Cmd::Enter => {
+                    let command = self.ignition.selected_command();
+                    let selected = state.rack_state.selected;
+                    info!(self.log, "Sending {command:?} to {selected}");
+                    self.popup = None;
+                    Some(Action::Ignition(selected, command))
+                }
```

Comment:
Could you add a comment here saying to keep this in sync with the impl in `overview.rs`? (I guess there might be some differences over time which mean that the code can't directly be shared.)

Reply:
Certainly - added comments to both modules in 8610a7ea